css-brochure-form/styles.css

```diff
@@ -0,0 +1,63 @@
+form {
+  width: 50%;
+  margin: auto;
+  margin-left: 6.8in;
```

Comment:
This line should not be necessary. Also, you should avoid the `in` unit, it represents inches, which is a pretty non-standard measurement